scalableminds/webknossos
# Add new volume layer to existing annotations

Currently, the feature looks like this:
![image](https://user-images.githubusercontent.com/2486553/144261440-b84634c1-3499-4cc1-a6e7-f8423a85799f.png)


### URL of deployed dev instance (used for testing):
- https://addvolumelayer.webknossos.xyz/

### Steps to test:
- create a new annotation for a dataset with segmentation layer
  - use the new "Add Volume Annotation Layer" button in the left sidebar to add another layer
  - the annotation should be saved and reloaded automatically
  - the new layer should appears and should allow editing
- create a new annotation for a dataset with segmentation layer, but choose "Create empty layer" in the extended create-modal
  - use the new "Add Volume Annotation Layer" button in the left sidebar to add another layer
  - now, the annotation should have two layers (both should not have fallback data)
  - --> ideally, the modal would also allow adding a layer with fallback, but this is not really feasible right now, as we remove all data layers without volume tracings during the model initialization (see code)
- create a skeleton-only annotation for a dataset with segmentation layer
  - add a new volume annotation layer
  - this time, the modal should allow you to choose which fallback layer you want to have
- in general, do some monkey testing by painting in different layers and switching between them

### Issues:
- fixes #5855

------
